Update interfaces/web-nfc.idl

autofoolip opened the pull request:
This PR was automatically created by a bot.

Before merging, please check that any tests that depend on the updated IDL files still work.

If additional changes are needed, please manually create another PR based on this one.

See the [README](https://github.com/web-platform-tests/wpt/blob/master/interfaces/README.md) for how the IDL files in this directory are used.

<hr>

Source: https://github.com/tidoust/reffy-reports/blob/36d3c3d/ed/idl/web-nfc.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/681927624

stephenmcgruer commented:
The associated idlharness file `web-nfc/idlharness.https.window.js` should be updated alongside this to remove its call to `new NDEFWriter()`.

autofoolip committed:
Update interfaces/web-nfc.idl

Source: https://github.com/tidoust/reffy-reports/blob/36d3c3d/ed/idl/web-nfc.idl
Build: https://travis-ci.org/tidoust/reffy-reports/builds/681927624

stephenmcgruer reviewed (approved):
Nevermind, the interface removal was reverted and this is just a dict change now.
